.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This screenshot is from PR 19. It shows how prettier broke up this line. I find it awful. I'm not sure what rule causes this, but can you please make prettier not to do it?
![kép](https://user-images.githubusercontent.com/38769170/132959969-e8fc8b1d-7069-4b33-bbde-10592d34ca64.png)